production/ksonnet/README.md

```diff
@@ -17,6 +17,11 @@ In your config repo, if you don't yet have the directory structure of Tanka set
 $ mkdir config && cd config/
 $ tk init
 $ tk env add loki --namespace=loki
+# notice that k8s_master_node_ip could not be localhost or 0.0.0.0
```

Comment:
Why not? From Tanka‘s side there is no limitation (however `0.0.0.0` is not even a valid IP)

Reply:
It only work when k8s apiserver address is localhost, for most case, i do not think anyone do this setting.

Reply:
I do for example! When using `k3s` or `k3d`, you have the apiServer locally at `127.0.0.1:6443`.

Furthermore I have the feeling saying that it _could not_ be something makes users think that there is some kind of limitation, which there certainly isn‘t.

Reply:
yep, i limit it in k8s